packages/core/test/Form_test.js

```diff
@@ -63,6 +63,7 @@ describeRepeated("Form common", (createFormComponent) => {
 
     it("should render errors if schema isn't object", () => {
       const props = {
+        showErrorList: "top",
```

Comment:
there actually is no test for top or bottom. we would need a snapshot, but I dont see any

Reply:
Core doesn't use snapshots. You can update things to verify the position

Reply:
Hmmm, All of the other themes use snapshots and display errors. Perhaps testing in each of the themes is usable instead? Search for the "help and error display" tests and add ones for `false` and `bottom`?

Reply:
in `validate_test.js` at line 63, adding the following will verify that the errors are on top
```
          expect(node.childNodes[0].className).to.eql('panel panel-danger errors');
```

To test if it is at the bottom you can probably copy the existing test and switch out that line with:

```
          // The last child node is the submit button so the one before it will be the error list
          expect(node.childNodes[node.childNodes.length - 2].className).to.eql('panel panel-danger errors');
```